js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetNode.tsx

```diff
@@ -212,9 +215,17 @@ export const AssetNodeMinimal = ({
   const queuedRuns = liveData?.unstartedRunIds.length;
   const inProgressRuns = liveData?.inProgressRunIds.length;
 
+  const topTagsPresent = facets?.has(AssetNodeFacet.UnsyncedTag);
+  const bottomTagsPresent = facets?.has(AssetNodeFacet.KindTag);
+
+  const paddingTop = topTagsPresent ? 32 : facets?.size === 0 ? 4 : facets ? 12 : height / 2 - 52;
+  const nodeHeight = facets
+    ? Math.max(38, height - (topTagsPresent ? 32 : 14) - (bottomTagsPresent ? 32 : 14))
+    : 86;
```

Comment:
Might be nice to make constants for these numbers so that we know what they represent